src/server/game/Time/GameTime.h

```diff
@@ -53,11 +53,11 @@ class TC_GAME_API GameTime
 
     private:
         // Game Time
-        time_t _startTime;
-        time_t _gameTime;
-        uint32 _gameMSTime;
-        std::chrono::system_clock::time_point _gameTimeSystemPoint;
-        std::chrono::steady_clock::time_point _gameTimeSteadyPoint;
+        time_t _startTime = time(nullptr);
```

Comment:
I agree with Shauren, but i'm only a peasant. Call me if you come to an agreement.